Packs/Logzio/Integrations/Logzio/Logzio.py

```diff
@@ -211,7 +212,21 @@ def fetch_incidents(client, last_run, search, severities, first_fetch_time):
         next_run["last_fetch"] = max(start_query_time, time.time() - ONE_HOUR)
     raw_events = client.fetch_triggered_rules(search=search, severities=severities,
                                               start_time=start_query_time)
-    for event in raw_events.get("results", []):
+
+    # Filter events if any of the filter tags are in an event
+    tag_filtered_events = []
+    if filter_tags:
+        for event in raw_events.get("results", []):
```

Comment:
Instead of going twice on raw_events it think it better to create a function that check if the event have those tags 
Than on the regular loop call the function
False - move to the next event 
True - continue with the loop logic